src/node.cpp

```diff
@@ -338,7 +338,10 @@ Node* Node::unserialize(MegaClient* client, string* d, node_vector* dp)
     hasLinkCreationTs = MemAccess::get<char>(ptr);
     ptr += sizeof(hasLinkCreationTs);
 
-    for (i = 6; i--;)
+    const bool syncable = MemAccess::get<bool>(ptr);
```

Comment:
from cppreference:  "bool - type, capable of holding one of the two values: true or false. The value of sizeof(bool) is implementation defined and might differ from 1."

Reply:
I didn't know it could be different from 1 but do you see a problem here?

Reply:
Ah got you. We'd have to take up more extension bytes of course. Ok, hm. Maybe best to write out an 8bit integer then. I'll change that.